pydantic/tools.py

```diff
@@ -42,6 +43,7 @@ def parse_file_as(
     encoding: str = 'utf8',
     proto: Protocol = None,
     allow_pickle: bool = False,
+    json_loads: Callable[[str], Any] = json.loads,
```

Comment:
good catch, could we add a test for this too.

Reply:
Thanks. I've added a test. And, it's just as well you suggested one as I had forgotten to change how we call `load_file`.